server/service/src/stocktake/update.rs

```diff
@@ -196,7 +196,7 @@ fn generate_stock_line_update(
             total_before_tax: 0.0,
             total_after_tax: 0.0,
             tax: None,
-            number_of_packs: quantiy_change,
+            number_of_packs: delta,
```

Comment:
number of packs should never be negative, StockOut and StockIn determines the direction, so this should still be quantity_change  ?

Reply:
[1e4018a](https://github.com/openmsupply/open-msupply/pull/642/commits/1e4018aa314ec1bba6457ad71d8afddda8601bc3) Quick test~

Can add more if you want more tests 😄

Reply:
🙏 That looks good for now